.github/workflows/scripts/publish-sdkman.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# This is script for publishing scala on SDKMAN.
+# Script resolves the latest stable version of scala and then send REST request to SDKMAN Vendor API.
+# It's releasing and announcing the release of scala on SDKMAN.
+#
+# Requirement:
+#   - the latest stable version of scala should be available in github atrifacts
```

Comment:
```suggestion
#   - the latest stable version of scala should be available in github artifacts
#   - environment variable SDKMAN_KEY containing the Consumer-Key to use to perform calls to SDKMAN API
#   - environment variable SDKMAN_TOKEN containing the Consumer-Token to use to perform calls to SDKMAN API
```

Reply:
3 more comments : 
1) on line 18 one should read "URL doesn't exist" instead of "URL not exists"
2) both `curl` calls on lines 23 and 34 should use same indentations for arguments `-X`, `-H`, etc.
3) both `curl` calls shoud be followed by a failure test `if [[ $? -ne 0 ]]; then ... fi`

Reply:
@michelou  Do you think `set -eu` is insufficient? and should we manually handle the error from curl execution?

Reply:
@lwronski It's just a matter to know if the error occurs with `-X POST` or with `-X PUT`. [`set -eu`](https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html) doesn't help to this respect.